models/localMacros.yaml

```diff
@@ -18,7 +18,7 @@ macros:
       - number_of_days
     value: "
       {% if warehouse.DatabaseType() == \"bigquery\" %}
-        {% if !(end_time|isnil) %} date_diff(date('{{end_time.Format(\"2006-01-02 15:04:05\")}}'), date({{column}}), day) <={{number_of_days}}
+        {% if !(end_time|isnil) %} date_diff(date('{{end_time.Format(\"2006-01-02 15:04:05\")}}'), date({{column}}), day)
```

Comment:
Why are we removing comparison with `number_of_days` ?

Reply:
we are not removing it, at line 23 we have another occurrence of `<= {{number_of_days}}`, so the one removed in line 21 was just making it occur twice and causing the project to fail.